src/test/custom_node_buffersqueue_test.cpp

```diff
@@ -51,11 +51,16 @@ TEST(CustomNodeBuffersQueue, GetAllBuffers) {
     }
 }
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-function"
+
 static void getBufferAndReturn(BuffersQueue& buffersQueue) {
```

Comment:
Why this can't be removed?

Reply:
If it is unused then we can remove it

Reply:
Ok I thought it was left because we will use it in the future.